src/hooks/useLoadClientScript.ts

```diff
@@ -38,6 +39,11 @@ export function useLoadClientScript(props: NostoScriptProps) {
       window.nostojs(api => api.setAutoLoad(false))
     }
 
+    if (!loadScript) {
```

Comment:
Is this change backward-compatible? 
If the existing merchants update their plugin, this may cause issues on their store

Reply:
yes, the default value of loadScript is true